include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -273,6 +273,13 @@ typedef struct OrtCUDAProviderOptions {
   int do_copy_in_default_stream;
 } OrtCUDAProviderOptions;
 
+typedef struct OrtOpenVINOProviderOptions {
```

Comment:
>typedef struct OrtOpenVINOProviderOptions  [](start = 0, length = 42)

Please, provide documentation for the type as a whole.

Reply:
Done, I added it for the existing CUDA structure as well